lib/parser.dart

```diff
@@ -666,7 +666,7 @@ class _Parser {
         //     '}'
         _next();
 
-        var name;
+        dynamic name;
```

Comment:
This is kind of a fun one - I wanted to avoid changing any behavior so I only gave real types where I could. This surfaced the fact that `_peekIdentifier()` must _always_ return `false` for this code path. If it doesn't then `name` would be an `Identifier`, since `identifier` always returns a non-null `Identifier`. However the `StyletDirective` constructor expects `name` to be a `String`, so the only value that works is `null`. There is almost certainly a logic bug here that was masked by overly dynamic code, but I don't know exactly what it is to fix it... The null safe Dart migration won't be fun for this package...

Reply:
:man_shrugging:

This isn't making things worse in any case.